plugins/jobs/src/js/components/Breadcrumbs.js

```diff
@@ -10,7 +11,9 @@ export default function Breadcrumbs({ jobPath, jobName, jobInfo, children }) {
   let breadcrumbParts = [
     <Breadcrumb key={"Jobs"} title="Jobs">
       <BreadcrumbTextContent>
-        <Link to={"/jobs/overview/"}>{"Jobs"}</Link>
+        <Link to={"/jobs/overview/"}>
+          <Trans render="span">Jobs</Trans>
```

Comment:
See above comment. I've been avoiding the span render in similar BreadcrumbTextContent components.